### ISSUE TITLE
doc/core/man/twistd.1 seems outdated

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4619]
|[<img alt="jafd's avatar" src="https://avatars.githubusercontent.com/u/301073?s=50" width="50" height="50">](https://github.com/jafd)| @jafd reported|
|-|-|
|Trac ID|trac#4619|
|Type|defect|
|Created|2010-07-29 14:48:33Z|

There is a couple of things in man 1 twistd which can be seen as typos, and some describe things which apparently are not there anymore.

First, there is some tautology. A fragment near --rundir is as follows:

```
       -d, --rundir <directory>
              Change to a supplied directory before running (default: .).
       -u, --uid <uid>
              The uid to run as (default: don't change).
       -g, --gid <gid>
              The gid to run as (default: don't change).
```

and then a bit further:

```
       -g, --plugin <plugin name>
              Read config.tac from a plugin package, as with -y.
       --syslog
              Log to syslog instead of a file.
       -u, --uid <uid>
              The uid to run as.
       -g, --gid <gid>
              The gid to run as.
```

First, I believe the --plugin parameter went away at some point, as twistd doesn't display it altogether. Second, -g --plugin conflicts with -g --gid. And third, --uid and --gid are described in two places. 

Attachments:

* [twistd-man.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b15/b1529f8a5f99bf4c2e9e2bbc2a4163c6aa5fc263/7710d53cdd49eb997321f3f3bb7ddc6701fcf0ca.patch) (1489 bytes) - added by thijs on 2011-05-27 13:23:40Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4619 4619
type__defect defect
reporter__jafd jafd
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__manpage documentation, manpage
time__1280414913000000 1280414913000000
changetime__1306866065000000 1306866065000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for reporting this, jafd, and thanks for working up a patch, thijs.

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `assigned`|
|-|-|

[#5125](https://github.com/twisted/twisted/issues/5125) was a duplicate but merged, so I'll just apply the rest of the patch which removes the --plugin option.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32006]) Apply twistd-man.patch: Remove old --plugin option and update dates.

Author: thijs
Reviewer: jesstess
Fixes: [#4619](https://github.com/twisted/twisted/issues/4619)
